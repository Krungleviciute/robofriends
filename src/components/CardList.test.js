import React from "react";
import { create } from "react-test-renderer";
import CardList from "./CardList";

describe("CardList Component Tests", () => {
  it("should check if Card List component renders", () => {
    const mockRobots = [
      {
        id: 1,
        name: "Dalia Henry",
        username: "DH",
        email: "dalia@henry.com",
      },
    ];
    const cardList = create(<CardList robots={mockRobots} />);
    expect(cardList.toJSON()).toMatchSnapshot();
  });
});
