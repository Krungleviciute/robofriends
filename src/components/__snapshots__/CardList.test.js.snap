// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList Component Tests should check if Card List component renders 1`] = `
<div>
  <div
    className="tc bg-green dib br3 pa3 ma2 grow bw2 shadow-5"
  >
    <img
      alt="robo profile"
      src="https://robohash.org/1?size=200x200"
    />
    <div>
      <h2>
        Dalia Henry
      </h2>
      <p>
        dalia@henry.com
      </p>
    </div>
  </div>
</div>
`;
